socialhome/templates/base.html

```diff
@@ -50,9 +50,11 @@
                               <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Logout" %}</a>
                           </li>
                       {% else %}
+                          {% if is_open_for_signup %}
```

Comment:
Where did you find this - it works? If you turn signups on, the link will show? Can you write a test for it? As you can see from the coverage, the part inside this `if` is now not executed in tests, which means either our tests run with signups disabled, or this code doesn't work.

The latter could be because the template engine by default doesn't raise on undefined variables, which is kind of confusing, should probably change that.